README.md

```diff
@@ -1,26 +1,34 @@
 # Beamsplitter
 
-Beamsplitter is Coherence's asset conversion platform built on Solana. This repo is for the on-chain program, if you are looking for Beamsplitter's client you should go to [insert client repo here](https://www.youtube.com/watch?v=xuCO7-DLCaA)
+Beamsplitter is Coherence's cryptocurrency ETF program on [Solana](https://solana.com/). This monorepo contains the on-chain program as well as the client sdk for interacting with Beamsplitter. It uses, [Anchor](https://project-serum.github.io/anchor/), a framework that simplifies development on Solana. You can use the program to buy, sell, request transfer of listed etf tokens. The sdk allows the same functionality
```

Comment:
“You can use the program to buy, sell, AND request transfer of listed etf tokens”

Reply:
You cannot, the program is for registration, construction, and deconstruction of ETFs. The readme is still incorrect but that is outside scope this PR.